### PR TITLE
Refactor cosmic-core pom.xml

### PR DESCRIPTION
**Pull request by michaelandersen**
build result on cosmic parent: 

```
mvn clean install -Pdevelopment
```

```
[INFO] Apache CloudStack Plugin - Storage Volume CloudByte Provider SUCCESS [  7.100 s]
[INFO] Apache CloudStack Plugin - Storage Volume sample provider SUCCESS [  4.817 s]
[INFO] Apache CloudStack Plugin - SNMP Alerts ............. SUCCESS [ 10.712 s]
[INFO] Apache CloudStack Plugin - Syslog Alerts ........... SUCCESS [  9.772 s]
[INFO] Apache CloudStack Plugin - Network VXLAN ........... SUCCESS [ 12.453 s]
[INFO] Apache CloudStack Plugin - GloboDNS ................ SUCCESS [ 18.073 s]
[INFO] Apache CloudStack Plugin - Quota Service ........... SUCCESS [ 15.323 s]
[INFO] Apache CloudStack Framework - Spring Module ........ SUCCESS [ 12.454 s]
[INFO] Apache CloudStack Secondary Storage Controller ..... SUCCESS [  5.880 s]
[INFO] Apache CloudStack Client UI ........................ SUCCESS [ 14.503 s]
[INFO] Apache CloudStack Console Proxy - RDP Client ....... SUCCESS [ 17.697 s]
[INFO] Apache CloudStack Console Proxy .................... SUCCESS [  1.328 s]
[INFO] Apache CloudStack Console Proxy - Server ........... SUCCESS [  8.321 s]
[INFO] Apache CloudStack Framework - QuickCloud ........... SUCCESS [  0.827 s]
[INFO] Apache CloudStack .................................. SUCCESS [  0.417 s]
[INFO] Apache CloudStack .................................. SUCCESS [  0.417 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 07:06 min (Wall Clock)
[INFO] Finished at: 2016-01-22T12:13:59+01:00
[INFO] Final Memory: 111M/1645M
[INFO] ------------------------------------------------------------------------

```

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -23,159 +23,24 @@
...
-    <developerConnection>scm:git:https://git-wip-us.apache.org/repos/asf/cloudstack.git</developerConnection>
-  </scm>
-  <issueManagement>
-    <system>jira</system>
-    <url>https://issues.apache.org/jira/browse/CLOUDSTACK</url>
-  </issueManagement>
 
   <prerequisites>
     <maven>3.0.4</maven>
   </prerequisites>
```
Comment (wilderrodrigues) [inline pom.xml]: Should we remove it? We should use the latest maven, which is 3.3.9

Cheers,
Wilder

Comment (michaelandersen) [inline pom.xml]: let's add the issuemanagement and scm when we have something to define there.
willing to move to 3.3.9 but will need to test the build again

Comment (borisroman) [inline pom.xml]: You can remove it, so it will always take the latest.

Comment (wilderrodrigues) [inline pom.xml]: Ah... sorry, My comment was not clear. The "should we remove it?" is related to the ```prerequisites``` entry.

Yes, remove and build again. no problem. :)

Comment (michaelandersen) [inline pom.xml]: ah ok. will force push now

**Comment by borisroman** [comment]
LGTM :+1:

**Pull request merged**